dpsim-models/CMakeLists.txt

```diff
@@ -4,31 +4,15 @@ configure_file(
 )
 
 set(MODELS_INCLUDE_DIRS
-	${EIGEN3_INCLUDE_DIR}
-	${SPDLOG_INCLUDE_DIR}
+	${CMAKE_CURRENT_SOURCE_DIR}/include
+	${CMAKE_CURRENT_BINARY_DIR}/include
```

Comment:
this change seems weird

Reply:
This are actually two unrelated changes:

- The spdlog and Eigen include directories are not required here as they are gathered by the libraries (line 14-15)
- We add the included directories local to this target